web/client/product/app.jsx

```diff
@@ -12,10 +12,14 @@ const LocaleUtils = require('../utils/LocaleUtils');
 
 const startApp = () => {
     const ConfigUtils = require('../utils/ConfigUtils');
-
+    const CoordinatesUtils = require('../utils/CoordinatesUtils');
+    const projectionsDefs = require('./utils/projectionsDefs');
+    projectionsDefs().forEach((proj) => {
+        window.proj4.defs(proj.code, proj.def);
+        CoordinatesUtils.addProjections(proj.code, proj.extent, proj.worldExtent);
```

Comment:
This should be in Openlayers map tool. Is not properly a CoordinateUtil, so move the internal code inside OL map.